templates/docs/examples/patterns/code-snippet/_script.js

```diff
@@ -0,0 +1,34 @@
+function setupCodeSnippetOptions(dropdown) {
+  var dropdownLabel = dropdown.querySelector('span');
+  var targetMenu = document.getElementById(dropdown.getAttribute('aria-controls'));
+  var options = [].slice.call(targetMenu.querySelectorAll('a'));
```

Comment:
We shouldn't target options via tag name. Using class name would be better (we know it as it's part of the pattern), but I guess even better we could check for aria-controls attribute maybe? `[aria-controls]`? Because we rely on this attribute for the functionality anyway.

Reply:
good point, updated